deeplearning4j-nlp-parent/deeplearning4j-nlp/src/main/java/org/deeplearning4j/models/paragraphvectors/ParagraphVectors.java

```diff
@@ -1211,6 +1211,10 @@ public Builder useAdaGrad(boolean reallyUse) {
         /**
          * This method defines whether negative sampling should be used or not
          *
+         * PLEASE NOTE: If you're going to use negative sampling, you might want to disable HierarchicSoftmax, which is disabled by default
```

Comment:
The comment doesn't make sense on his own: why disable HierarchicSoftmax if it's disabled by default? You mean enabled.